src/app/core/models/state.ts

```diff
@@ -0,0 +1,8 @@
+export class State {
+    state: string;
```

Comment:
Should be name instead of State.state. I think that would make it clear.

Reply:
Good point